app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -235,7 +236,19 @@ const MigrationsCompletedList = ({
                           ),
                           !isMissingMapping && (
                             <ListView.InfoItem key={`${plan.id}-infraMappingName`}>
-                              {plan.infraMappingName}
+                              <a
+                                href="#"
+                                onClick={e => {
+                                  e.preventDefault();
+                                  filterMappingsListOnTransitionAction({
+                                    filterTypeId: 'name',
+                                    filterValue: plan.infraMappingName
```

Comment:
There may not be a way to solve this, but because our filters match on partial strings, `infraMappingName` is not always going to be unique enough to obtain the exact mapping we want.

For example, you could have a plan created with mapping `test`, but also have a mapping named `test #2`